aws/network.tf

```diff
@@ -1,7 +1,3 @@
-resource "aws_eip" "bosh" {
-  vpc = true
-}
-
```

Comment:
This static IP is required by the microbosh so that it can communicate with AWS in order to spin up new VMs, upload stemcells etc.